src/libstd/fs.rs

```diff
@@ -635,6 +635,22 @@ impl FileType {
     /// Test whether this file type represents a symbolic link.
     #[stable(feature = "file_type", since = "1.1.0")]
     pub fn is_symlink(&self) -> bool { self.0.is_symlink() }
+
+    /// Test whether this file type represents a block device.
+    #[unstable(feature = "more_filetypes", reason = "recently added API")]
+    pub fn is_block_device(&self) -> bool { self.0.is_block_device() }
```

Comment:
`more_filetypes` sounds very general. Do you think `file_types_posix` is a better name? (Just to suggest something too rather than just nitpicking!)